src/main/java/sirius/kernel/commons/URLBuilder.java

```diff
@@ -63,7 +63,8 @@ public URLBuilder(@Nonnull String protocol, @Nonnull String host) {
     /**
      * Adds a path part to the url.
      * <p>
-     * Once the first parameter has been added, the path can no longer be modified.
+     * Once the first parameter has been added, the path can no longer be modified. If the path itself contains
+     * parameters the questionMark {@link Monoflop} will be toggled.
```

Comment:
Not really sure that we need to explain this logic in the JavaDoc, especially making reference to an internal private property.

Reply:
I agree, we should not put internal behaviour into javadoc. Maybe add a comment to the parameter instead, saying that it can contain query parameter if they are properly started by '?'?

Reply:
I changed the JavaDoc a little to meet your expectations.